script/bench.rb

```diff
@@ -49,13 +49,15 @@ def run(command, opt = nil)
       system(command, out: $stdout, err: :out)
     end
 
-  exit unless exit_status
+  abort("Command '#{command}' failed with exit status #{$?}") unless exit_status
 end
 
 begin
   require 'facter'
 rescue LoadError
   run "gem install facter"
+  # Facter requires CFPropertyList, but doesn't install it.
```

Comment:
Could you point me to the source of this dependency? I don't have this Gem installed on my machine and I'm still able to install the `facter` gem